server/models/Game.js

```diff
@@ -21,11 +22,12 @@ const gameSchema = new mongoose.Schema({
     mechanics: [String],
     designers: [String],
     artists: [String],
-    publisher: String,
-    categories: [{
-        type: Schema.Types.ObjectId,
-        ref: "Category"
-    }]
+    publishers: [String],
+    categories: [String],
```

Comment:
This might not be a great way of doing it. We will talk about this tomorrow

Reply:
Yeah, there's likely a better way to implement this.  Just a quick solution for now.  A new PR will include new virtuals to populate the categories and mechanics we pulled from BGA.